kanidmd/daemon/src/opt.rs

```diff
@@ -102,33 +142,29 @@ enum KanidmdOpt {
     #[clap(name = "configtest")]
     /// Test the IDM Server configuration, without starting network listeners.
     ConfigTest(CommonOpt),
-    #[clap(name = "backup")]
-    /// Backup the database content (offline)
-    Backup(BackupOpt),
-    #[clap(name = "restore")]
-    /// Restore the database content (offline)
-    Restore(RestoreOpt),
-    #[clap(name = "verify")]
-    /// Verify database and entity consistency.
-    Verify(CommonOpt),
     #[clap(name = "recover_account")]
-    /// Recover an account's password
+    /// Recover an account's password (must be run offline)
```

Comment:
Technically all of these have to .....

Reply:
Weeeeelllllll the db things don't, but sure, removed. 😄